src/js/Content/Features/Store/Common/FAddToCartNoRedirect.js

```diff
@@ -35,7 +35,18 @@ export default class FAddToCartNoRedirect extends Feature {
                     return null;
             }
 
-            return document.forms[formName];
+            let formEl = document.forms[formName];
+
+            // Special handling for bundles on wishlist
+            if (onWishlist && !formEl && fnName === "addBundleToCart") {
```

Comment:
The second condition is unnecessary isn't it?

Reply:
Just in case Steam fixes it, since `formEl` gets reassigned below.